src/main/scala/allawala/chassis/core/module/ChassisModule.scala

```diff
@@ -5,32 +5,49 @@ import javax.inject.Named
 import akka.actor.ActorSystem
 import akka.event.{Logging, LoggingAdapter}
 import akka.stream.ActorMaterializer
+import allawala.chassis.auth.module.AuthModule
 import allawala.chassis.config.model.BaseConfig
 import allawala.chassis.config.module.ConfigModule
 import allawala.chassis.health.module.HealthModule
 import allawala.chassis.http.module.HttpModule
 import allawala.chassis.http.service.AkkaHttpService
+import allawala.chassis.i18n.module.I18nModule
 import allawala.chassis.util.module.UtilModule
-import com.google.inject.{AbstractModule, Module, Provides, Singleton}
+import com.google.inject.{AbstractModule, Provides, Singleton}
 import net.codingwell.scalaguice.ScalaModule
 
 import scala.concurrent.ExecutionContext
 
-
-class ChassisModule extends AbstractModule with ScalaModule {
+abstract class ChassisModule extends AbstractModule with ScalaModule {
   override def configure(): Unit = {
     install(new ConfigModule)
     install(new HttpModule)
+    bindHealthModule()
```

Comment:
why did you move these to a seperate method that is doing exactly this?

Reply:
So that the implementing services can overwrite as needed